### ISSUE TITLE
Hook up change payment account

### ISSUE THREAD

**Issue by andymckay-limited-access** [mozilla/payments-ui#291]
Subscribe to a product, go to subscriptions and click on change payment account. Nothing happens. Presumably there's other bugs for this in progress at the moment and this bug is not needed :)

**Comment by muffinresearch** [comment]
Yep, this was still to do. Not sure if there was another bug. Also the UI  has now changed too - there's now a drop-down instead of the link. 

That said here we might want a new view like before because it might not be great to try and cram paypal + cards into the one drop-down - currently the new dropdown is only for cards.
